runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -855,7 +855,10 @@ J9::Compilation::compileRelocatableCode()
 bool
 J9::Compilation::compilePortableCode()
    {
-   return self()->fej9()->inSnapshotMode();
+   return (self()->fej9()->inSnapshotMode() ||
+	   (compileRelocatableCode() &&
```

Comment:
I had suggested it because of a conceptual integrity issue. It'd be possible to have a Portable SCC in place, but the default front end would return false, which maybe in the context of a compilation is fine, but if you were to ask that question outside, you might get the wrong answer. As such I thought it best to just disallow anyone from using the default front end to ask that question.

Reply:
ok, then we need that FATAL_ASSERT to `TR_J9ServerVM` as well.

Reply:
Also, I think you have to do `self()->compileRelocatableCode()`.